src/main/scala/strawman/collection/SortedMap.scala

```diff
@@ -12,7 +12,9 @@ trait SortedMap[K, +V]
 
 trait SortedMapOps[K, +V, +CC[X, Y] <: SortedMap[X, Y] with SortedMapOps[X, Y, CC, _], +C <: SortedMap[K, V]]
   extends MapOps[K, V, Map, C]
-     with SortedOps[K, C] {
+     with SortedOps[K, C, SortedSet] {
+
+  def sortedIterableFactory = SortedSet
```

Comment:
Yes. This is a side-effect of `SortedOps` being shared by sorted sets and maps. I was also baffled by this at first but it does make sense. A sorted map is sorted by key. If you assume a standard `Ordering` on tuples, it degrades nicely to a sorted set. The ordering for `(K,V)` with unique `K` values is the same and with `K` already being unique, so is `(K,V)`.